test/middleware/spec.back-link.js

```diff
@@ -176,13 +184,23 @@ describe('Back Links', function () {
 
     it('permits whitelisting of steps from a separate form instance', function () {
         req.get.withArgs('referrer').returns('http://example.com/referrer');
+        req.baseUrl = '/referrer';
```

Comment:
I have explicitly assigned the correct value to `req.baseUrl` in all cases to illustrate that the stub with 'referrer' will not create a baseUrl and to avoid assumptions about tests.